src/js/apps/globals/app-frame_app.js

```diff
@@ -84,9 +103,13 @@ export default App.extend({
     });
   },
   showAppNav() {
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    if (!currentUser.can('program-admin')) topNavMenu.remove(topNavMenu.at(0)); // UGLY
```

Comment:
add an id that can be a string so you can do `topNavMenu.remove('program-admin');`